jekyll/_cci2/configuration-reference.md

```diff
@@ -1010,7 +1010,7 @@ CircleCI treats tag and branch filters differently when deciding whether a job s
 1. For a branch push unaffected by any filters, CircleCI runs the job.
 2. For a tag push unaffected by any filters, CircleCI skips the job.
```

Comment:
Are we still keeping these lines?

Reply:
Oooh, good catch. Forgot about the configuration reference. My vote is to remove for consistency. :)

Reply:
🔪 them

Reply:
This is the most violent thing I've seen in a PR! Congratulations!